apps/site/next-data/generators/blogData.mjs

```diff
@@ -39,12 +34,6 @@ const getFrontMatter = (filename, source) => {
   // all = (all blog posts), publish year and the actual blog category
   const categories = [category, `year-${publishYear}`, 'all'];
 
-  // we add the year to the categories set
```

Comment:
Why we removing these categories? :eyes:

Reply:
nono, these are still being added anyways to the `blogCategories` array, I'm just doing it later: https://github.com/nodejs/nodejs.org/pull/7618/files/7004629e8ff0dfcdc7a57109d5132fe34ef301c3#diff-5ddcf76b865830c001627fef7abd06107b3eafeb210d21737305db2416a279beR95-R98